<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_MainWin</class>
 <widget class="QMainWindow" name="Form_MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_browseFolder">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse folder of header files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_folderSelected">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder selected</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_outputGen">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>651</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_genSigs">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate CFFI signatures</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SDK_API_define</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sdkApiDefine">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>471</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_browseFolder</sender>
   <signal>clicked()</signal>
   <receiver>Form_MainWin</receiver>
   <slot>browse_folder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_genSigs</sender>
   <signal>clicked()</signal>
   <receiver>Form_MainWin</receiver>
   <slot>generate_sigs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_folder()</slot>
  <slot>generate_sigs()</slot>
 </slots>
</ui>
